win32ss/gdi/ntgdi/freetype.c

```diff
@@ -5,7 +5,7 @@
  * PURPOSE:         FreeType font engine interface
  * PROGRAMMERS:     Copyright 2001 Huw D M Davies for CodeWeavers.
  *                  Copyright 2006 Dmitry Timoshkov for CodeWeavers.
- *                  Copyright 2016-2017 Katayama Hirofumi MZ.
+ *                  Copyright 2016-2018 Katayama Hirofumi MZ.
```

Comment:
Unless you do a substantial revision (e.g. 50% of the lines were modified in a non-trivial way) you don't really need to update the Copyright; the year of the first publication will do just fine.